zebra/zebra_rib.c

```diff
@@ -1103,7 +1103,8 @@ static void rib_process(struct route_node *rn)
 					   ROUTE_ENTRY_CHANGED);
 			new_fib = best;
 		} else {
-			best = rib_choose_best(new_selected, re);
+			if (!CHECK_FLAG(re->flags, ZEBRA_FLAG_NEVER_SELECT))
```

Comment:
would it be better to stick this check into rib_choose_best() itself? just thinking that that might encapsulate the decision

Reply:
I thought about that and decided I wanted it right where we use the ZEBRA_FLAG_FIB_OVERRIDE since this is the opposite of that one.